app/_examples/client-component/page.tsx

```diff
@@ -1,29 +1,12 @@
-'use client'
+"use client";
 
-// TODO: Duplicate or move this file outside the `_examples` folder to make it a route
-
-import { createClientComponentClient } from '@supabase/auth-helpers-nextjs'
-import { useEffect, useState } from 'react'
+// import useTasks from "@/__backengine__/hooks/useTasks";
 
+// TODO: Duplicate, move this file outside the `_examples` folder and add your table to make it a route
 export default function ClientComponent() {
-  const [todos, setTodos] = useState<any[]>([])
-
-  // Create a Supabase client configured to use cookies
-  const supabase = createClientComponentClient()
-
-  useEffect(() => {
-    const getTodos = async () => {
-      // This assumes you have a `todos` table in Supabase. Check out
-      // the `Create Table and seed with data` section of the README 👇
-      // https://github.com/vercel/next.js/blob/canary/examples/with-supabase/README.md
-      const { data } = await supabase.from('todos').select()
-      if (data) {
-        setTodos(data)
-      }
-    }
-
-    getTodos()
-  }, [supabase, setTodos])
+  // this assumes you have a table called "tasks" in Backengine
+  // const { tasks } = useTasks();
```

Comment:
accidentally commented out?

Reply:
have to comment out otherwise the build fails (this hook doesn't exist it's just an example of how to use a generated hook)